src/schema.ts

```diff
@@ -147,23 +147,56 @@ type Topic {
   supplementaryResources: [Resource]
 }
 
+type FrontpageSubjects {
+  category: String
+  subjects: [String]
```

Comment:
This should return [Subjects]. So we can have a query like this:
```gql
{
  frontpage {
    categories {
      name 
      subjects {
        name
      }
    }
  }
}
```
 Make sure subjects is fetched with a loader.